src/main/scala/org/ergoplatform/nodeView/history/ErgoHistory.scala

```diff
@@ -314,6 +314,23 @@ object ErgoHistory extends ScorexLogging {
 
     repairIfNeeded(history)
 
+    // temporary hack which is injecting nipopow proof to the database to make it possible to bootstrap with
+    // nipopows + utxo set snapshot soon after 5.0.13 release
+    // todo: remove after height 1,096,693 on the mainnet
+    val bestHeaderHeight = history.headersHeight
+    if (bestHeaderHeight > 1054000 && bestHeaderHeight < 1096693) { //history.readPopowProofBytesFromDb().isDefined){
+      history.popowProofBytes(
+        history.P2PNipopowProofM,
+        history.P2PNipopowProofK,
+        Some(ModifierId @@ "25a11667e38e62412522c062d90b073afd9ed9551080ff4e0a67d1757ce18b98")) match {
```

Comment:
Please create a named constant with a description where this id come from.

Reply:
done